packages/bp/src/core/config/botpress.config.ts

```diff
@@ -22,7 +22,7 @@ export interface DialogConfig {
    * Interval before a session's context expires.
    * e.g. when the conversation is stale and has not reached the END of the flow.
    * This will reset the position of the user in the flow.
-   * @default 2m
+   * @default 25m
```

Comment:
The bot config [`timeoutInterval`](https://github.com/botpress/botpress/blob/master/packages/bp/src/core/config/bot.config.ts#L68) (default to 5min) has precedence over the one on the Botpress config. See: https://github.com/botpress/botpress/blob/master/packages/bp/src/core/dialog/sessions/expiry.ts#L20

This means that changing this value will have no real effect unless you remove the default timeout from your bot configs.

Reply:
Good catch, updated that one also :)